sass/layout/_headers.scss

```diff
@@ -59,20 +75,37 @@
     }
 
```

Comment:
For the `.channel-header__info .channel-header__description` selector, the height should be:
```
.channel-header__info .channel-header__description {
    height: 24px;
}
```

And can we add 4px margin on the left of the `.header-placeholder` selector?
```
.channel-header__info .channel-header__description .header-placeholder {
    margin-left: 4px;
}
```